build.gradle

```diff
@@ -1414,10 +1414,12 @@ project(':group-coordinator') {
     implementation project(':coordinator-common')
     implementation libs.jacksonDatabind
     implementation libs.jacksonJDK8Datatypes
+    implementation libs.lz4
     implementation libs.metrics
     implementation libs.hdrHistogram
     implementation libs.re2j
     implementation libs.slf4jApi
+    implementation libs.hash4j
```

Comment:
I suppose that we need to update the license file too.

Reply:
Yes, I added it to LICENSE-binary and `python committer-tools/verify_license.py` can pass. Do I need to modify other files? Thanks.

Reply:
This is fine. Thanks.